src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/FunctionalTests.cs

```diff
@@ -165,6 +165,28 @@ public static IEnumerable<object[]> GetDataForJson()
             });
         }
 
+        public static IEnumerable<object[]> GetDataForStu3ToR4()
+        {
+            var data = new List<string[]>
+            {
+                new[] { @"CapabilityStatement" },
+                new[] { @"CodeSystem" },
+                new[] { @"Observation" },
+                new[] { @"OperationDefinition" },
+                new[] { @"OperationOutcome" },
+                new[] { @"Parameters" },
+                new[] { @"Patient" },
+                new[] { @"StructureDefinition" },
+                new[] { @"ValueSet" },
+            };
+            return data.Select(item => new[]
+            {
+                item[0],
```

Comment:
Now that only the first element is needed, maybe just define a `List<string>` instead of `List<string[]>` could be simpler.

Reply:
Thanks, refined it.